Gemfile

```diff
@@ -33,6 +32,10 @@ gem "omniauth-gitlab"
 
 gem "rack-cors", "~> 1.0.1"
 
+# Supported DBs
+gem "mysql2", "= 0.4.7", group: :mysql
+gem "pg", "~> 0.20.0", group: :postgres
```

Comment:
I think we need to document how to setup the `bundle install` for each db case, no? This will require both postgres and mysql packages to be installed when `bundle install` happens. 

For development is fine but for production might be an issue.

Reply:
> This will require both postgres and mysql packages to be installed when bundle install happens.

I agree, more documentation should be put for this 👍

Reply:
👍